AnnoyancesFilter/sections/css_extended.txt

```diff
@@ -96,6 +96,7 @@ facebook.com,facebookcorewwwi.onion#%#(function() { var observer=new MutationObs
 eva.ru##div:has(> div > .subscribe-form)
 finanzen.net##.teaser > .box:has(> .box-headline:contains(WhatsApp Newsletter))
 finanzen.net##.col-sm-4 > .box:has(> .box-headline:contains(WhatsApp Newsletter))
+xakep.ru##.widget_custom_html:has(> .widget-inner > .textwidget > #mc_embed_signup > form[action*="list-manage.com/subscribe"])
```

Comment:
why not just `.theiaStickySidebar > div.widget_custom_html:has(> div > div > div#mc_embed_signup)`?
`form[action*="list-manage.com/subscribe"]` is redundant

Reply:
I agree, including both is redundant. Either one is enough.